client/state/selectors/get-jetpack-connect-redirect-after-auth.js

```diff
@@ -0,0 +1,20 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { getAuthorizationRemoteQueryData } from 'state/jetpack-connect/selectors';
+
+/**
+ * Returns redirect_after_auth provided as part of Jetpack Connect authorization.
+ *
+ * @param  {Object}  state Global state tree
+ * @return {?String}       Redirect URL
+ */
+export default function getJetpackConnectRedirectAfterAuth( state ) {
```

Comment:
Might be able to make the name more descriptive here? `getJetpackConnectRedirectUrl()`?

Reply:
Heh, if only it were that easy:

https://github.com/Automattic/wp-calypso/blob/eedc6dd87e05654830f86a41d824e217dbc493fe/client/state/jetpack-connect/reducer/schema.js#L51-L52

I'm not entirely sure what the difference is between these, but I agree the name could be improved here… I'll give it a think.

Reply:
mm I see. Fine to leave as is for now if necessary, since we aren't making things any worse with this name, and we can probably improve later.